contracts/resolvers/OwnedResolver.sol

```diff
@@ -14,7 +14,7 @@ import "./profiles/TextResolver.sol";
  * address.
  */
 contract OwnedResolver is Ownable, ABIResolver, AddrResolver, ContentHashResolver, DNSResolver, InterfaceResolver, NameResolver, PubkeyResolver, TextResolver {
-    function isAuthorised(bytes32 node) internal override view returns(bool) {
+    function isAuthorised(bytes32) internal override view returns(bool) {
```

Comment:
Is the argument not to be used? If so, isn't the convention to comment out like (`/* bytes32 */`)?